CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.3.0 - 2022-07-12
```

Comment:
```suggestion
## 0.3.0 - 2022-08-23
```
This date should also be updated given the difference (if it was only a couple days it'd be fine but over a month is too much).